### PR TITLE
fixed splide.

### PR DESCRIPTION
**Pull request by myniqx**
image

### CODE REVIEW

File: packages/ui/components/ModelMedia/ModelMedia.tsx [hunk 1]
```diff
@@ -62,15 +62,17 @@ export const ModelMedia = <T extends FieldValues = FieldValues>({
...
-              <ModelMedia
-                model={newModel}
-                name={name}
-                isEditing={isEditing}
-                isChangingMedia={isChangingMedia}
-                toggleChangingMedia={toggleChangingMedia}
-                setValue={setValue}
-                endpoint={endpoint}
-              />
+              <Center w={'400px'}>
```
Comment (7alip) [inline packages/ui/components/ModelMedia/ModelMedia.tsx:65]: Should it be `maxW={400} w={full}`?

Comment (myniqx) [reply packages/ui/components/ModelMedia/ModelMedia.tsx:65]: w=full didnt work. splideslide doesnt give room modelmedia to grow.

Comment (7alip) [reply packages/ui/components/ModelMedia/ModelMedia.tsx:65]: Well, I saw narrow empty model image section in #1960 then, there I needed to add extra w={'full'}

**Review by 7alip — APPROVED** [review]
Thanks

**Pull request merged**